data/sql_setup/prepare_schedule_a.sql

```diff
@@ -214,3 +214,16 @@ create trigger nml_sched_a_after_trigger after insert or update
 drop trigger if exists nml_sched_a_before_trigger on disclosure.nml_sched_a;
 create trigger nml_sched_a_before_trigger before delete or update
     on disclosure.nml_sched_a for each row execute procedure ofec_sched_a_delete_update_queues(:START_YEAR_AGGREGATE);
+
+CREATE OR REPLACE FUNCTION insert_sched_master() RETURNS TRIGGER AS $$
+DECLARE
+    child_table_prefix TEXT = TG_ARGV[0];
+    child_table_name TEXT;
+BEGIN
+    child_table_name = child_table_prefix || get_partition_suffix(new.two_year_transaction_period);
```

Comment:
Another thing about this trigger: we still need to reference the view row corresponding to the record that was inserted, and make sure it is present in the view to begin with.  Otherwise, the row will either be based only on one of the underlying source tables, or potentially made up of almost all `NULL` values if we're not careful when pulling from the view.